src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/FrameworkExtensionTest.php

```diff
@@ -636,4 +650,47 @@ private function assertVersionStrategy(ContainerBuilder $container, Reference $r
             $this->assertEquals($format, $versionStrategy->getArgument(1));
         }
     }
+
+    private function assertCachePoolServiceDefinitionIsCreated(ContainerBuilder $container, $name, $type, $defaultLifetime, $namespaceArgumentIndex = null)
```

Comment:
@trousers are you a bot? If so, I fear we get spammed. We use fabbot for automatic code reviews and I really doubt we want an other bot we can't configure here.
About the comments, I'd really like to get comments on the actual behavior of the code before getting more on the length of variables.